vitest.base.unit.config.ts

```diff
@@ -20,7 +20,7 @@ export default defineConfig({
     ],
     reporters: process.env.GITHUB_ACTIONS
       ? ["verbose", "hanging-process", "github-actions"]
-      : ["verbose", "hanging-process"],
+      : [process.env.USE_BASIC_REPORTER === "true" ? "basic" : "verbose", "hanging-process"],
```

Comment:
It's good and aligned to what we already have before changing to `verbose` because other team members want to see all the tests which are passing as well not just the one which are failing. 

We may change this flag to `TEST_COMPACT_OUTPUT`  or something similar for clarity and to remember.